internal/ignition/ignition.go

```diff
@@ -1510,7 +1511,7 @@ func (g *installerGenerator) modifyPointerIgnitionMCP(poolName string, ignitionS
 	return "", errors.Errorf("machine config pool %s was not found", poolName)
 }
 
-func (g *installerGenerator) writeSingleHostFile(host *models.Host, baseFile string, workDir, serviceBaseURL, bootReporter string, authType auth.AuthType) error {
+func (g *installerGenerator) writeSingleHostFile(host *models.Host, baseFile string, workDir string) error {
```

Comment:
Any clue why `serviceBaseURL, bootReporter and  authType` are in the signature???

Reply:
Were not used so i removed them